App.tsx

```diff
@@ -474,11 +476,24 @@ const BaseApp: React.FunctionComponent<{
     );
   }
 
+  const linking = {
+    prefixes: [prefix, FormatCenterWebsite(avalancheCenterId as AvalancheCenterWebsiteSchema) + '/observations/#/view/'],
+    config: {
+      screens: {
+        Observations: {
+          screens: {
+            observation: 'observations/:id',
```

Comment:
Here is where you could add `observations/#/view/:id` so that the navigation container associates the path to this screen. Not 100% positive about this though

Reply:
Both of the following link to an individual observation:

https://nwac.us/observations/#/view/observations/fb5bb19a-2b89-4c9c-91d2-eb673c5ab877
https://nwac.us/observations/#/observation/fb5bb19a-2b89-4c9c-91d2-eb673c5ab877

The former (which is what you're writing about) overlays the observation on the list view, while the latter just renders it stand-alone. We might be able to support both, but if we can't I think what Yulia has now (support for the latter) is likely the better option as it maps more cleanly to the screen we're sending users to.

Reply:
Ah, actually ... looks like @yuliadub supports only the latter, with `https://nwac.us/observations/#/view/` configured as the accepted prefix and `observations/fb5bb19a-2b89-4c9c-91d2-eb673c5ab877` the path.

Reply:
In the future we will likely want to support `https://nwac.us/` as the prefix but if we're only trying to route obs pages for now, /shrug

Reply:
oooohhhh okay that makes more sense I see what's going on now. Thanks for clarifying

Reply:
yeah I am also not 100% sure of this mapping and how it will work. From what I tested on expo, it seems like the way I have it set up makes sense....but need to try it on an actual non expo build to see what works -- once we get that working will appreciate any help figuring it out if it doesnt work